crates/admin/src/cluster_controller/scheduler.rs

```diff
@@ -326,15 +397,15 @@ impl<T: TransportConnect> Scheduler<T> {
 /// Represents the scheduler's observed state of the cluster. The scheduler will use this
 /// information and the target scheduling plan to instruct nodes to start/stop partition processors.
 #[derive(Debug, Default, Clone)]
-struct ObservedClusterState {
-    partitions: BTreeMap<PartitionId, ObservedPartitionState>,
-    alive_nodes: BTreeMap<PlainNodeId, GenerationalNodeId>,
-    dead_nodes: BTreeSet<PlainNodeId>,
-    nodes_to_partitions: BTreeMap<PlainNodeId, BTreeSet<PartitionId>>,
+pub struct ObservedClusterState {
+    pub partitions: BTreeMap<PartitionId, ObservedPartitionState>,
+    pub alive_nodes: BTreeMap<PlainNodeId, GenerationalNodeId>,
```

Comment:
The reason why those sets are separate atm is because it makes some checks a bit cheaper (is any of my nodes dead/unhealthy, for example). Might be a micro optimization though.

Reply:
I suspected that's the case - don't worry about it; I hadn't come across a scenario where that was useful but probably just missed it.